src/main/resources/static/simple.soy

```diff
@@ -95,6 +103,15 @@
   {/call}
 
   {call aui.form.textareaField}
+    {param id: 'errorMessage.COMMITTER_EMAIL_DOMAIN' /}
```

Comment:
I don't believe this error message customization is working because `COMMITTER_EMAIL_DOMAIN` does not match the YACC error type `COMMITTER_EMAIL_REGEX`.

The "DOMAIN" type might be from a prior iteration of the code? I think `COMMITTER_EMAIL_REGEX` is good for the error type... the error message just needs to be updated to reflect this.

Can you verify this?